Adds a few more pulsar star station trait reports

Rhials opened the pull request:
## About The Pull Request

This adds some more station-trait dependent Pulsar Star reports, much like the "Clown Planet" report triggered by the bananium shipments trait.

For those of you not up to speed - When pulsar star (no meaningful threat report is provided) rolls, and a relevant station trait is active, the generic report is replaced with one pertaining to that trait.

It's better to just check the code changes to get a full picture, but I'll give some summaries on what the individual reports are here:

**Assistant Gimmick Outfits** - "Grey Sky", and a warning to monitor the recently emboldened assistants on-station.

**Intern** - The intern has forgotten to actually include the report in his message. What a goof!

**Ion Stormfront** - Oh dear, the electromagnetic interference is messing with the report broadcast.
![image](https://github.com/tgstation/tgstation/assets/28870487/ea7060b5-177e-4760-a079-9f61650aca27)

**Endless Snowstorm** - Ice Giant, Too much snow, we can't give an actual report. Stay safe!

**Wise Cow Invasion** - Cow Planet. [We're gonna go fast and we're gonna go far with a steady driving bass and a rhythm guitar.](https://www.youtube.com/watch?v=BPqPYszbSFI)
## Why It's Good For The Game

Seeing the goofy clown planet message always made me think "dang, this feels like underutilized material" and can give command some prompting on what to tell the crew beyond "yeah we don't know what threat level it is". Maybe it'll prompt someone to do a gimmick based on the report? Who knows! It's just an added dash of flavor at the end of the day.

Admittedly, some are a bit silly, but also rare (requiring both a specific station trait and the 8% pulsar star chance) so I think its fine.
## Changelog
:cl: Rhials
add: Adds some more station-trait dependent pulsar star reports. Keep an eye on that roundstart command report!
/:cl:

Rhials committed:
adds more pulsar reports

Rhials committed:
why do I even care about this

Rhials committed:
unnecessarily hostile message

MrMelbert commented on code/controllers/subsystem/dynamic/dynamic.dm:
```diff
@@ -380,6 +380,31 @@ SUBSYSTEM_DEF(dynamic)
+			advisory_string += "Your sector's advisory level is Grey Sky. Our sensors detect abnormal activity among the assistants assigned to your station. We advise you to closely monitor the Tool Storage, Bridge, Tech Storage, and Brig for gathering crowds or petty thievery."
+			return advisory_string
+
+		if(HAS_TRAIT(SSstation, STATION_TRAIT_WISE_COWS))
```
Maybe at this point it should be a `/datum/station_trait` proc? like `/datum/station_trait/proc/get_advisory_note`

Then we can just

```
for(var/datum/station_trait/trait as anyhting in shuffle(SSstation.traits))
    var/trait_string = trait.get_advisory_note()
    if(trait_string)
        advisory_string += trait_string 
        break

Admiral-Gigaton commented:
Feel like Grey sky could be changed to pertain more to rain, and shit? Grey clouds... or high tide

zoomachina commented:
Will this result in 500% more pulsar star shifts?

Rhials commented:
> Will this result in 500% more pulsar star shifts?

No, but it will result in 500% more pulsar star command report variety!

DrAmazing343 commented:
Grey Sky sounds about right. Grey Storm would sound a bit violent/like Ion Storm, or something. Grey Rain is a bit tacky because it's like ooooOOOOOoooo the greyshits are RAINING FROM THE SKYYYY. I dunno, Grey Sky invokes the imagery just right.

Time-Green commented:
Do the thing that melbert said. Make sure it only overwrites the pulsar announcements (you can check this on the trait). The priority on the traits can be random (if you want)

Rhials committed:
still need to remove the otherwise unused traits

Rhials committed:
wow

while I was testing this I forgot to set the plsar report chance to 100% so I really just got lucky with the 8% chance roll on that particular round being a pulsar star round wowo wow

Rhials committed:
Merge remote-tracking branch 'upstream/master' into no-threat-level

Rhials committed:
artifact from merge conflict oopsies!
